### ISSUE TITLE
Missing properties/types in interfaces.

### ISSUE THREAD

**Issue by IgorSzymanski** [RobinBuschmann/sequelize-typescript#486]
Would it be possible to add missing properties/types to some interfaces?
Like:

`skip` property missing in `Model.save()` method.
```ts
interface InstanceSaveOptions extends FieldsOptions, LoggingOptions, ReturningOptions, SearchPathOptions {

        /**
         * If true, the updatedAt timestamp will not be updated.
         *
         * Defaults to false
         */
        silent?: boolean;

		/** THIS IS MISSING
		 * Array of fields ignored during validation.
		 */
		skip?: string[];
    }
```
The only reason I'm not making a PR is because I'm not entirely sure where that property belongs. `InstanceSaveOptions`, `FieldsOptions` or `ICreateOptions`.

From the docs (can't find it in v4 docs, but it still works with v4):
https://sequelize.readthedocs.io/en/v3/api/instance/
> Name: [options.skip], Type: Array, Description: An array of strings. All properties that are in this array will not be validated

Another example I found is this interface:
```ts
export interface IIncludeOptions extends IBaseIncludeOptions {
    /**
     * The model you want to eagerly load
     */
    model?: typeof Model;
    /**
     * The association you want to eagerly load. (This can be used instead of providing a model/as pair)
     */
    association?: IIncludeAssociation;
    /**
     * Load further nested related models
     */
    include?: Array<typeof Model | IIncludeOptions>;
    /**
     * If true, only non-deleted records will be returned. If false, both deleted and non-deleted records will
     * be returned. Only applies if `options.paranoid` is true for the model.
     */
	paranoid?: boolean;
	
	nested?: boolean;
}
```
Property `association` should also accept `string` type, so I think it should be something like:
```ts
association?: IIncludeAssociation | string;
```
Still not sure if that's how it's supposed to be fixed.

From the docs:
http://docs.sequelizejs.com/manual/tutorial/models-usage.html#eager-loading

> You can also include by alias name by specifying a string that matches the association alias:
```js
User.findAll({ include: ['Instruments'] }).then(users => {
  console.log(JSON.stringify(users))

  /*
    [{
      "name": "John Doe",
      "id": 1,
      "createdAt": "2013-03-20T20:31:45.000Z",
      "updatedAt": "2013-03-20T20:31:45.000Z",
      "Instruments": [{
        "name": "Toothpick",
        "id": 1,
        "createdAt": null,
        "updatedAt": null,
        "userId": 1
      }]
    }]
  */
})
```

Thanks!

By the way, I love this library. I gave up TypeORM for this and I don't regret.
Keep up the good work!

**Comment by RobinBuschmann** [comment]
Hey @IgorSzymanski, thanks for contributing and the nice words as well  🙂

`InstanceSaveOptions` seem to be the right place. But this interface is part of `@types/sequelize` package. So you would need to PR against this one.

The other example confuses me a bit. You say `association` should accept a `string`, but in your example you are using `include: ['Instruments']`.  Do you initially meant `include` here? If so the `IIncludeOptions` are the right place, exactly.

Hope this helps.

**Comment by IgorSzymanski** [comment]
I copied the wrong example from the docs. It should've been:

```js
User.findAll({ include: [{ association: 'Instruments' }] }).then(users => {
  console.log(JSON.stringify(users))

  /*
    [{
      "name": "John Doe",
      "id": 1,
      "createdAt": "2013-03-20T20:31:45.000Z",
      "updatedAt": "2013-03-20T20:31:45.000Z",
      "Instruments": [{
        "name": "Toothpick",
        "id": 1,
        "createdAt": null,
        "updatedAt": null,
        "userId": 1
      }]
    }]
  */
})
```

**Comment by RobinBuschmann** [comment]
@IgorSzymanski Sry for the late reply. [This](https://github.com/RobinBuschmann/sequelize-typescript/blob/master/lib/interfaces/IIncludeOptions.ts#L18) should be the right place for enabling `string` for `association`

**Comment by RobinBuschmann** [comment]
Fixed by #518
